commons/zenoh-protocol/src/network/oam.rs

```diff
@@ -21,6 +21,12 @@ pub mod flag {
     pub const Z: u8 = 1 << 7; // 0x80 Extensions    if Z==1 then an extension will follow
 }
 
+pub mod id {
```

Comment:
For style consistency, just `use super::OamId` and then replace the full path expansion`crate::transport::oam::OamId` with `OamId`.